src/scene/scene_lua.inl

```diff
@@ -28,73 +28,115 @@ namespace openspace {
 
 namespace {
 
-void applyRegularExpression(lua_State* L, std::regex regex, std::vector<properties::Property*> properties, int type) {
+void executePropertySet(properties::Property* prop, lua_State* L) {
+    prop->setLuaValue(L);
+    //ensure properties are synced over parallel connection
+    std::string value;
+    prop->getStringValue(value);
+    /*OsEng.parallelConnection().scriptMessage(
+    prop->fullyQualifiedIdentifier(),
+    value
+    );*/
+}
+
+template <class T>
+properties::PropertyOwner* findPropertyOwnerWithMatchingGroupTag(T* prop,
+const std::string tagToMatch)     {
+    properties::PropertyOwner* tagMatchOwner = nullptr;
+    properties::PropertyOwner* owner = prop->owner();
+    
+    if (owner != nullptr) {
+        std::vector<std::string> tags = (std::vector<std::string>)owner->tags();
+        for (std::string currTag : tags) {
```

Comment:
I'll pass on this one. It looks like a smoother way to do it but if the old way works, I don't want to change it and potentially break functionality.